tools/pot/openvino/tools/pot/graph/special_operations.py

```diff
@@ -65,6 +65,9 @@
     {'type': 'TopK'}
 ]
 
+# TODO: Add attributes to GraphTransformer hw_config
+TYPES_TO_QUANTIZABLE_PORTS = {'LSTMSequence': [0, 1, 4, 5], 'GRUSequence': [0, 1, 3, 4]}
```

Comment:
I think that `LSTMCell` was lost.

Reply:
Yes, because we quantize only Sequences in last version